week3/homework/src/actions.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const low = require('lowdb');
+const FileSync = require('lowdb/adapters/FileSync');
+const adapter = new FileSync('db.json');
+const db = low(adapter);
+const nanoid = require('nanoid');
+const state = { todos: [] };
+db.defaults(state).write();
+
+function getToDos() {
+  return db.get('todos');
+}
+function getTodo(id) {
+  return getToDos().find({ id });
+}
+function get() {
+  return getToDos().value();
+}
+function getItem(id) {
+  return getTodo(id).value();
+}
+
+function done(id) {
+  return db
+    .get('todos')
```

Comment:
You already created a function to read the todos from the database, so you could replace `db.get('todos')` with `getToDos()`. The same note applies on all functions below.

Reply:
Thank you for feedback and for your advices.